### ISSUE TITLE
persistent_workspaces naming is inconsistent.

### ISSUE THREAD

**Issue by sw1nn** [Alexays/Waybar#2193]
The key in the config file for `persistent_workspaces` has an inconsistent format compared to the other keys.

All the other keys in the config file are `kebab-case` whereas `persistent_workspaces` is snake case. 

The fix is easy, but if I did a PR would you expect to support the old naming as well to avoid breaking existing configs.

**Comment by Alexays** [comment]
Yeah, you could make a PR while keeping the backward compatibility

**Comment by Alexays** [comment]
Resolved on master: the option was renamed to `persistent-workspaces` (kebab-case, consistent with the rest of the config) back in 2023; the old `persistent_workspaces` spelling was kept as a deprecated alias for a while and later removed. Please use `persistent-workspaces`. Closing as resolved.
